restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/SmsInterpreter.java

```diff
@@ -573,6 +574,7 @@ public void execute(final Object message) throws Exception {
             buffer.append(initialSessionSid.toString());
             final URI uri = URI.create(buffer.toString());
             builder.setUri(uri);
+            SmsVerb.populateAttributes(verb, builder);
```

Comment:
@jaimecasero to me it seems redundant to hide the preparation of action attributes in a method. Its only 5 lines of code and reduces the visibility of the full task here.

Reply:
i think this is a good effort on hiding specific verb stuff in a particular class. Right now the interpreter is already to big, and this way we reuse code for other interpreters.